lib/cli/commands/streamorders.ts

```diff
@@ -9,7 +9,7 @@ export const describe = 'stream order added, removed, and swapped events (DEMO)'
 
 export const builder = {
   existing: {
-    description: 'should return existing orders',
+    description: 'returns existing orders',
```

Comment:
Maybe "whether to return existing orders" ?

Reply:
Overlooked that it's a boolean. Fixed.